apps/web-giddh/src/app/contact/payment-aside/payment-aside.component.ts

```diff
@@ -161,4 +271,192 @@ export class PaymentAsideComponent implements OnInit {
+        this.store.dispatch(this._companyActions.getAllIntegratedBankInCompany(this.companyUniqueName));
+    }
+
+    /**
+     *To select bank event
+     *
+     * @param {*} event selected bank object
+     * @memberof PaymentAsideComponent
+     */
+    public selectBank(event: IOption): void {
+        this.selectedBankUrn = event.value;
+        if (event) {
+            this.requestObjectTogetOTP.urn = event.value;
+            this.requestObjectTogetOTP.bankType = event.label;
+            this._companyService.getAllBankDetailsOfIntegrated(this.companyUniqueName, this.requestObjectTogetOTP.urn).subscribe(response => {
+
+                if (response.status === 'success') {
+                    if (response.body.Status === 'SUCCESS') {
+                        this.totalAvailableBalance = response.body.effectiveBal;
+                    }
+                } else if (response.status === 'error') {
+                    this._toaster.errorToast(response.message, response.code);
+                }
+            })
+        }
+    }
+
+    /**
+        *To select bank event
+        *
+        * @param {*} event selected bank object
+        * @memberof PaymentAsideComponent
+        */
+    public bulkPayVendor(): void {
+        this._companyService.bulkVendorPayment(this.companyUniqueName, this.requestObjectTogetOTP).subscribe(response => {
+
+            if (response.status === 'success') {
+                this._toaster.successToast(response.body);
+            } else if (response.status === 'error') {
+                this._toaster.errorToast(response.message, response.code);
+            }
+        });
+    }
+
+
+
+
+    public getTotalAmount(selectedAccount: any[]): void {
+        this.totalSelectedAccountAmount = 0;
+        if (selectedAccount && selectedAccount.length) {
+            this.totalSelectedAccountAmount = selectedAccount.reduce((prev, cur) => {
+                return prev + cur.closingBalanceAmount;
+            }, 0);
+        }
+    }
+
+    /**
+     * To Prepare payment request object and API call
+     *
+     * @param {NgForm} form Form data
+     * @memberof PaymentAsideComponent
+     */
+    public payClicked(form: NgForm) {
+        this.isPayclicked = true;
+        this.prepareRequestObject();
+        this.bulkPayVendor();
+    }
+
+    public prepareRequestObject() {
+        this.requestObjectTogetOTP.bankPaymentTransactions = [];
+        this.requestObjectTogetOTP.totalAmount = String(this.totalSelectedAccountAmount);
+        this.selectedAccForBulkPayment.forEach(item => {
+            let transaction: BankTransactionForOTP = {
+                amount: '',
+                remarks: '',
+                vendorUniqueName: ''
+            };
+            transaction.amount = item.closingBalance.amount;
+            transaction.remarks = item.remarks;
+            transaction.vendorUniqueName = item.uniqueName;
+            this.requestObjectTogetOTP.bankPaymentTransactions.push(transaction);
```

Comment:
Do use proper camelCase
```suggestion
            this.requestObjectToGetOTP.bankPaymentTransactions.push(transaction);
```